### PR TITLE
Feat/is 455 text cards

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR introduces a new text card component for the homepage.

Component:
Screenshot 2023-09-15 at 10 08 29 AM

Figma:
https://www.figma.com/file/teqEgFdMSMqRVtw8jajlup/Website-Template?type=design&node-id=7216%3A122111&mode=design&t=EbVJmuEkS5ARGgZa-1

Live demo:
https://staging.duyfy15grdtiq.amplifyapp.com/

### CODE REVIEW

File: _sass/homepage/_text-cards.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  font-size: 1.25rem;
+}
+
+.textcards-card-section {
+  justify-content: center;
+  align-items: flex-start;
+  gap: 2rem;
+  width: 100%;
+  
+  @media screen and (max-width: 1023px) {
```
Comment (dcshzj) [inline _sass/homepage/_text-cards.scss]: nit: are we able to switch to the globally defined breakpoints introduced in #314?

File: _sass/homepage/_text-cards.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
+.textcards-section {
+  flex-direction: column;
+  align-items: center;
+
+  @media screen and (max-width: 768px) {
```
Comment (dcshzj) [inline _sass/homepage/_text-cards.scss]: We should be able to switch to the breakpoint-aware padding and margin utilities now with #314 merged.

File: _sass/homepage/_text-cards.scss (hunk 1 above)
Comment (alexanderleegs) [reply _sass/homepage/_text-cards.scss]: Have swapped to the values in https://github.com/isomerpages/isomerpages-template/pull/313/commits/706b3472e15e47673c22c6300af3e073f5050449! Was unable to use the variables, they broke for remote build :(

File: _sass/homepage/_text-cards.scss (hunk 2 above)
Comment (alexanderleegs) [reply _sass/homepage/_text-cards.scss]: Swapped in https://github.com/isomerpages/isomerpages-template/pull/313/commits/706b3472e15e47673c22c6300af3e073f5050449!

**Review by dcshzj — APPROVED** [review]

**Pull request merged**